static/bidder-params/smarthub.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "SmartHub Adapter Params",
-  "description": "A schema which validates params accepted by the SmartHub adapter",
+  "title": "Attekmi Adapter Params",
+  "description": "A schema which validates params accepted by the Attekmi adapter",
   "type": "object",
   "properties": {
     "partnerName": {
       "type": "string",
-      "description": "SmartHub unique partner name",
+      "description": "Attekmi unique partner name",
```

Comment:
I suggest changing these three occurrences to "Attekmi Smarthub"

Reply:
Our setup instructions only have "Attekmi partner name", without "Smarthub"